README.md

```diff
@@ -58,10 +58,6 @@ Although in the global scope, it is not available until after the `deviceready`
 
 - Only one media file can be played back at a time.
 
-- There are strict restrictions on how your application interacts with other media. See the [Microsoft documentation for details][url].
-
-[url]: http://msdn.microsoft.com/en-us/library/windowsphone/develop/hh184838(v=vs.92).aspx
-
```

Comment:
The link redirects to a redirect to a page that does not sound like the original one that was linked to